src/core/dfn.js

```diff
@@ -17,6 +20,10 @@ export function run() {
     if (dfn.dataset.dfnFor) {
       dfn.dataset.dfnFor = dfn.dataset.dfnFor.toLowerCase();
     }
+    if (!("noexport" in dfn.dataset)) {
+      dfn.dataset.export = "";
```

Comment:
Should we do `dfn.removeAttribute("data-export")` instead?

Reply:
Oh. I misunderstood.

Reply:
This is not removing an attribute, this is adding one.